apps/backend/db_patches/0162_AddStatusActionsLogsTables.sql

```diff
@@ -0,0 +1,27 @@
+DO
+$$
+BEGIN
+  IF register_patch('0162_AddStatusActionsLogsTables.sql', 'Farai Mutambara', 'Add Status Actions Logs', '2024-09-02') THEN
+    BEGIN
+       CREATE TABLE IF NOT EXISTS status_actions_logs(
+            status_actions_log_id serial PRIMARY KEY,
+            connection_id int NOT NULL,
+            action_id int NOT NULL,
+            email_status_action_recipient text DEFAULT NULL,
+            status_actions_by int REFERENCES users (user_id) ON UPDATE CASCADE,
```

Comment:
do we need to have this tight integrity? Will this affect the user deletion?

Reply:
Have removed this column since it is not necessary in this table.